jest.config.packages.js

```diff
@@ -141,7 +141,7 @@ module.exports = {
   // runner: "jest-runner",
 
   // The paths to modules that run some code to configure or set up the testing environment before each test
-  setupFiles: ['../../tests/setup.ts'],
+  // setupFiles: ['../../tests/setup.ts'],
```

Comment:
Why comment it out?

Reply:
We used the same pattern than for other configuration fields in this file:
- https://github.com/MetaMask/accounts/pull/55/files/54eb2a99f2b8bcf0396ac1cbe21621a8a70643fd#diff-67d7eaa7b8718c9f7f2fb10f5be717922c0175ecd68f4af4c3520e505580eb4bR146-R147